package.json

```diff
@@ -80,13 +80,15 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.6.6",
+    "@popperjs/core": "^2.11.6",
```

Comment:
Although we only explicitly import a type from `@popperjs/core`, needed as a peerdependency of `react-popper`